app/templates/views/dashboard/dashboard.html

```diff
@@ -1,10 +1,16 @@
+
+
 {% extends "withnav_template.html" %}
 
 {% from "components/table.html" import list_table, field, text_field, link_field, right_aligned_field_heading, hidden_field_heading, row_heading, notification_status_field, notification_carrier_field, notification_carrier_message_field %}
 {% from "components/ajax-block.html" import ajax_block %}
 
 {% block service_page_title %}
   Dashboard
+
+<script type="text/javascript" src="{{ asset_url('js/setTimezone.js') }}"></script>
```

Comment:
This placement is causing the script to be in the title 
![image](https://github.com/user-attachments/assets/189c266a-0a8d-484e-a5ee-f34fe0552fab)

Reply:
I'm not seeing js/setTimezone.js invoked

Reply:
On my local, I moved the script so that it can be invoked and was able to retrieve the cookies 
![image](https://github.com/user-attachments/assets/8407bf63-cfb0-46c4-a6c9-8ef37a09843c)

Reply:
I'm curious because you are not in Los Angeles.  I am not either (30 or 40 miles north).  I wonder how the browser decides where people are ... but still, pacific time is correct, so I guess it's fine.

Reply:
I moved it, and every works for me in my local (as usual).  There is a green debug message that will light up if the python gets invoked and reads the timezone from the cookie.